packages/annotations/src/annotations.ts

```diff
@@ -62,28 +62,52 @@ export function getAnnotationServiceUrl(
 export function searchAnnotations(
   requestOptions: IQueryFeaturesRequestOptions
 ): Promise<any /* make it a type */> {
+  if (!requestOptions.outFields) {
+    requestOptions.outFields = [
+      "OBJECTID",
+      "author",
+      "description",
+      "source",
+      "status",
+      "target",
+      "dataset_id",
+      "created_at",
+      "updated_at"
+    ];
+  }
+
   return queryFeatures(requestOptions).then(response => {
     const users: string[] = [];
 
-    if (response.features && response.features.length > 0) {
-      response.features.forEach(function(comment) {
-        if (users.indexOf(comment.attributes.author) === -1) {
-          users.push(comment.attributes.author);
-        }
-      });
-    }
+    // use .reduce()?
+    response.features.forEach(function(comment) {
+      if (users.indexOf(comment.attributes.author) === -1) {
```

Comment:
let's build up `data` in here and add the `id`s (and maybe later the relationships), so: 
```
const attributes = comment.attributes;
data.push({
  id: attributes.OBJECTID,
  type: 'annotation',
  attributes
});
if (include === 'users') {
  users.push(attributes.author);
}
```